lib/scope.js

```diff
@@ -164,6 +164,9 @@ function recursiveScanScope(path, bindings) {
 
         recursiveScanChild(path.get("body"), bindings);
 
+    } else if (namedTypes.TypeAlias.check(node)) {
```

Comment:
My only request would be to add a defensive check that `namedTypes.TypeAlias` is defined:
```js
} else if (namedTypes.TypeAlias &&
            namedTypes.TypeAlias.check(node)) {
```
I want to make it easier to load selective subsets of AST types (#57), and I would like this code to continue working even if the Flow types are not loaded.